projects/plugins/jetpack/extensions/blocks/ai-image/edit.js

```diff
@@ -69,14 +94,16 @@ export default function Edit( { attributes, setAttributes, clientId } ) {
 		};
 	}, [] );
 
-	const submit = () =>
+	const submit = () => {
+		setErrorMessage( '' );
 		getImagesFromOpenAI(
-			prompt,
+			prompt.trim() === '' ? PLACEHOLDER : prompt,
```

Comment:
I have some doubts about that, but I'll allow it.

Reply:
:)